stdlib/asyncio/sslproto.pyi

```diff
@@ -70,8 +89,14 @@ class _SSLProtocolTransport(transports._FlowControlMixin, transports.Transport):
     def write(self, data: bytes) -> None: ...
     def can_write_eof(self) -> Literal[False]: ...
     def abort(self) -> None: ...
+    if sys.version_info >= (3, 11):
```

Comment:
```suggestion
    if sys.version_info >= (3, 11):
        def get_write_buffer_limits(self) -> tuple[int, int]:
```

Reply:
I was confused about why this hadn't been flagged by stubtest -- looks like in the stub it's identical to the signature of `get_write_buffer_limits()` in the superclass, `asynico.transports._FlowControlMixin`. Happy to add the override here, though, since I agree it's a bit weird and counterintuitive to have the other three methods here but not this one.